cmp/viewmanager/View.ts

```diff
@@ -33,24 +47,25 @@ export class View<T extends PlainObject = PlainObject> {
         return this.info?.lastUpdated ?? null;
     }
 
-    get token(): string {
-        return this.info?.token ?? null;
+    get typedName(): string {
+        return `${this.model.typeDisplayName} '${this.name}'`;
     }
 
     static fromBlob<T>(blob: JsonBlob, model: ViewManagerModel): View<T> {
-        return new View(new ViewInfo(blob, model), blob.value);
+        return new View(new ViewInfo(blob, model), blob.value, model);
     }
 
-    static createDefault<T>(): View<T> {
-        return new View(null, {});
+    static createDefault<T>(model: ViewManagerModel): View<T> {
+        return new View(null, {}, model);
     }
 
     withUpdatedValue(value: Partial<T>): View<T> {
-        return new View(this.info, value);
+        return new View(this.info, value, this.model);
     }
 
-    constructor(info: ViewInfo, value: Partial<T>) {
+    constructor(info: ViewInfo, value: Partial<T>, model: ViewManagerModel) {
```

Comment:
Believe this should be `model: ViewManagerModel<T>`, but also curious why the backpointer to the model.  Looks like just for the `type `and `typedName`. Wonder if it would make sense to have a `getTypedName` helper that takes a view and a model or view and a type rather than having it be a top-level property on view itself